src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -305,6 +309,10 @@ private void markGivenCell(Cell cell) {
 		HighlightPainter painter = new DefaultHighlighter.DefaultHighlightPainter(
 				Color.green);
 
+		/* TODO - THIS MAY NOT BE THE BEST PLACE TO HAVE THIS SET BUT I NEED THIS TO MAKE PENCIL MARK WORKING
+		 * DO NOT FORGET REVISIT AND UPDATE ValueType logic */
+		cell.setType(ValueType.Given);
```

Comment:
There was a bug in the puzzle generator that was not setting the ValueType. I have resolved this and will commit to to my branch. Then this change will not be necessary. In the mean time, this shouldn't hurt anything, but we can remove it eventually.